runelite-client/src/main/java/net/runelite/client/plugins/keyremapping/KeyRemappingListener.java

```diff
@@ -156,7 +156,12 @@ else if (config.esc().matches(e))
 				}
 			}
 
-			if (mappedKeyCode != KeyEvent.VK_UNDEFINED)
+			if (plugin.isDialogOpen() && config.space().matches(e))
+			{
+				mappedKeyCode = KeyEvent.VK_SPACE;
+			}
+
+			if (mappedKeyCode != KeyEvent.VK_UNDEFINED && e.getKeyCode() != mappedKeyCode)
```

Comment:
You could add this to the code down in keyRelease too

Reply:
True